package.json

```diff
@@ -48,7 +48,7 @@
     "build:electronRenderer": "cd packages/apps-electron && NODE_ENV=production webpack --config webpack.renderer.config.js",
     "build:devElectronMain": "cd packages/apps-electron && webpack --config webpack.main.config.js",
     "build:electronMain": "cd packages/apps-electron && NODE_ENV=production webpack --config webpack.main.config.js",
-    "prepackElectron": "yarn build:electron && yarn postinstall:electron",
+    "prepackElectron": "yarn build && yarn build:electron && yarn postinstall:electron",
```

Comment:
Maybe rename this one to `build:release:electron` (For the sake of consistency). at the same time would need to remove the step for that in the workflow yml, otherwise it will be fired twice